File: 프로그래머스/lv0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.js
function solution(my_string) {
  const exceptStr = my_string.replace(/[^0-9]/g, ' ');
  const strToArray = exceptStr.split(' ');
  let answer = 0;
  strToArray.forEach((number) => {
    if (number !== '') {
      answer += Number(number);
    }
  })
  
  return answer;
}

File: 프로그래머스/lv0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.py
def solution(my_string):
    answer = 0
    
    tmp = ""
    for str in my_string:
        if str.isdigit():
            tmp += str
        else:
            if tmp:
                answer += int(tmp)
                tmp = ""
    if tmp:
        answer += int(tmp)
        tmp = ""
    return answer